lib/cmxl/fields/floor_limit_indicator.rb

```diff
@@ -0,0 +1,24 @@
+module Cmxl
+  module Fields
+    class FloorLimitIndicator < Field
+      self.tag = 34
+      self.parser = /(?<currency>[a-zA-Z]{3})(?<debit_mark>[DC]?)(?<amount>[\d|,|\.]{4,15})/i
+
+      def credit?
+        data['debit_mark'].empty? || data['debit_mark'] == 'C'
+      end
+
+      def debit?
+        data['debit_mark'].empty? || data['debit_mark'] == 'D'
+      end
+
+      def currency
+        data['currency']
+      end
```

Comment:
this should already be handled by: https://github.com/railslove/cmxl/blob/master/lib/cmxl/field.rb#L120

Reply:
true dat